ranger-drove/src/main/java/io/appform/ranger/drove/servicefinderhub/DroveServiceDataSource.java

```diff
@@ -45,6 +50,7 @@ public Collection<Service> services() {
         Preconditions.checkNotNull(mapper, "mapper has not been set for node data");
         return droveClient.services()
                 .stream()
+                .filter(serviceName -> !excludedServices.contains(serviceName))
```

Comment:
Revert to original implementation and filter after the service list is returned from source in servicefinderhub, client etc. Those are in our control and removes the burden to filter from data source implementers.

The other issue here is that, the interface doesn't dictate the implementation to even know or honor exclusions.  Let us not do exclusions like this.